x-pack/plugins/secops/server/lib/hosts/elasticsearch_adapter.test.ts

```diff
@@ -127,9 +149,9 @@ describe('hosts elasticsearch_adapter', () => {
       expect(data).toEqual(expected);
     });
 
-    test('it formats a host with a source of name, firstSeen, os, and version correctly', () => {
+    test('it formats a host with a source of name, firstSeen, lastBeat, os, and version correctly', () => {
```

Comment:
Are we keeping firstSeen? If not, then this should read:

```js
it formats a host with a source of name, lastBeat, os, and version correctly
```

As this doesn't test `firstSeen` anymore.

Reply:
Nice catch -- I've updated the wording and removed `firstSeen` from the other `ReadonlyArray<string>` fields as they're implicitly tested since `firstSeen` is coming off of an aggregate and not from `_source`.